util/codec/codec_test.go

```diff
@@ -530,7 +530,10 @@ func (s *testCodecSuite) TestBytes(c *C) {
 }
 
 func parseTime(c *C, s string) types.Time {
-	m, err := types.ParseTime(nil, s, mysql.TypeDatetime, types.DefaultFsp)
+	sc := &stmtctx.StatementContext{
```

Comment:
```suggestion
	sc := &stmtctx.StatementContext{TimeZone: time.UTC}
```
Please make the code style consistent, thanks

Reply:
refactored.